frontend/src/components/Header.js

```
@@ -0,0 +1,67 @@
+    logout();
+    navigate('/login');
+  };
+
+  return (
+    <header style={styles.header}>
+      <h1 style={styles.title}>Vim Game</h1>
+      <div style={styles.userInfo}>
+        {user ? (
+          <>
+            <span style={styles.username}>{user.username}</span>
+            <button onClick={handleLogout} style={styles.button}>ログアウト</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)